specs/wp-gutenberg-post-editor-spec.js

```
@@ -152,6 +158,16 @@ describe( `[${ host }] Gutenberg Editor: Posts (${ screenSize })`, function() {
 			);
 		} );
 
+		step( 'Can see the post tag in preview', async function() {
```

This should make this fail faster if it isn't saved in the post